agent/task.go

```diff
@@ -161,20 +161,20 @@ func (a *Agent) runPreTaskCommands(ctx context.Context, tc *taskContext) error {
 		tc.ranSetupGroup = true
 	}
 
-	taskGroup, err := tc.taskConfig.GetTaskGroup(tc.taskGroup)
+	pre, err := tc.taskConfig.GetPre(tc.taskGroup)
 	if err != nil {
 		tc.logger.Execution().Error(errors.Wrap(err, "fetching task group for pre-task commands"))
 		return nil
 	}
 
-	if taskGroup.SetupTask != nil {
-		tc.logger.Task().Infof("Running setup task for task group '%s'.", taskGroup.Name)
-		opts.failPreAndPost = taskGroup.SetupGroupFailTask
+	if pre.Commands != nil {
+		tc.logger.Task().Infof("Running setup task for task group '%s'.", pre.Name)
```

Comment:
We can just remove this log. It doesn't really serve any purpose, doesn't apply to non-task-group tasks, we don't log it for the `teardown_task`, and you can deduce the command block from the `block` field.

Reply:
done